api/api.pb.go

```diff
@@ -626,17 +658,17 @@ func (m *StorageResource) GetLastScan() *google_protobuf.Timestamp {
 // StoragePool groups different storage devices based on their CosType
 type StoragePool struct {
 	// ID pool ID
-	ID int32 `protobuf:"varint,1,opt,name=ID" json:"ID,omitempty"`
+	ID int32 `protobuf:"varint,1,opt,name=ID,json=iD" json:"ID,omitempty"`
```

Comment:
same question - is this backward compatible ?

Reply:
Adding a new tag would not break backward compatibility.